Source/PagesController.swift

```diff
@@ -30,6 +30,10 @@ import UIKit
 
   lazy var pages = Array<UIViewController>()
```

Comment:
Other array style is preferred `[UIViewController]()` 
Thanks to Elvis making me picky to details 
:)